### PR TITLE
Migrate subset of combined tests using solid mechanics to tensor mechanics

### PR DESCRIPTION
**Pull request by sapitts**
### Design information
No source code is modified in this commit

### Test Cases
A subset of the combined module regression tests are replicated in both tensor and solid mechanics during the migration transition, with both mechanics formulations pointing to the same gold file where possible.  In the regression tests using a single element I've switched the regression tests to use the generated mesh type rather than a mesh file.

Refs #8642

### CODE REVIEW

**Comment by sapitts** [comment]
@bwspenc Would you review this commit of the combined tests as you have time please?  thanks!

File: modules/combined/tests/creep_plasticity_combined/combined_creep_plasticity.i [hunk 1]
```diff
@@ -9,7 +9,6 @@
 #   further yielding during unloading. The initial yield stress (sigma_0) is prescribed
 #   as 20 with a plastic strain hardening of 100. The mesh is a 1x1x1 cube with symmetry
 #   boundary conditions on three planes to provide a uniaxial stress field.
-#   The temperature is held constant at 1000.
```
Comment (bwspenc) [inline modules/combined/tests/creep_plasticity_combined/combined_creep_plasticity.i]: A  couple of questions about this test:
1. It looks like you removed the heat transfer part of it.  Could these tests be moved to tensor_mechanics now? It's good to move any tests that we can out of modules/combined/tests.
2. Did the temperature not affect the results before?

File: modules/combined/tests/creep_plasticity_combined/combined_stress_prescribed.i [hunk 2]
```diff
@@ -0,0 +1,159 @@
+#
+# 1x1x1 unit cube with time-varying pressure on top face
+#
+# The problem is a one-dimensional creep analysis.  The top face has a
+#    pressure load that is a function of time.  The creep strain can be
+#    calculated analytically.  There is no lsh plasticity.
```
Comment (bwspenc) [inline modules/combined/tests/creep_plasticity_combined/combined_stress_prescribed.i]: Does this comment mean that there is no plasticity active under these loading conditions?  There is a plasticity model.

File: modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation.i [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 1
+  ny = 1
+  nz = 1
+[]
+
+[Variables]
+  [./temp]
```
Comment (bwspenc) [inline modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation.i]: You removed temperature from the other models.  Why not from this one?

File: modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation_sm.i [hunk 4]
```diff
@@ -13,19 +13,22 @@
...
+#          a  = 3e-26  (creep coefficient)
+#          ef = 0.01   (displacement)
+#          t  = 2160.0    (time)
 #
-# The solution computed is very close to the exact solution.  This test is not a
-#     correct representation of the problem since it does not set the initial
-#     condition for the displacement and the stress.  Currently (1 March 2011),
-#     no capability exists for setting the initial condition of stress.  Until
-#     that is available, some error will be inherent in the solution.
+#    such that the analytical solution is computed to be 2.9518e3 Pa
```
Comment (bwspenc) [inline modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation_sm.i]: I'd like to talk for a bit about this.  If what was deleted in that comment is still true, we should probably leave that comment.

**Review by bwspenc — COMMENTED** [review]
It looks pretty good.  I'd just like to discuss the few things I noted before we merge this.

File: modules/combined/tests/creep_plasticity_combined/combined_stress_prescribed.i (hunk 2 above)
Comment (sapitts) [inline modules/combined/tests/creep_plasticity_combined/combined_stress_prescribed.i]: Thanks for catching this;  I'll clarify the comments to indicate that the plasticity model should not be active in this case (the yield stress is 1e30 MPa, so we shouldn't hit any plasticity during this tests).

File: modules/combined/tests/creep_plasticity_combined/combined_creep_plasticity.i (hunk 1 above)
Comment (sapitts) [inline modules/combined/tests/creep_plasticity_combined/combined_creep_plasticity.i]: Sure--I'd be glad to move these tests to tensor mechanics.  The analytical solution from the paper does not include temperature, so the temperature was held constant to avoid making an impact.  Should I move the solid mechanics tests to solid mechanics or just keep them here temporarily?

The isotropic plasticity model only uses the temperature if a yield stress function is provided (here one is not), and the creep model uses the temperature in the activation term, which we've essentially cancelled out in these input file by setting the activation energy Q to 0 to match the journal paper.

File: modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation.i (hunk 3 above)
Comment (sapitts) [inline modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation.i]: I should have.  Thanks for catching this too

File: modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation_sm.i (hunk 4 above)
Comment (sapitts) [inline modules/combined/tests/creep_plasticity_combined/combined_stress_relaxation_sm.i]: In solid mechanics I've seen other input files with an option in solid model for an initial stress (ironically it's the stress_relaxation2 that I've removed in this commit above).  We have the option for initial stress in tensor mechanics (see some of Andy's regression tests for the jacobian testing and/or borehole) or the option to use the extra_stress class Larry committed a bit ago.

**Comment by sapitts** [comment]
@bwspenc I've repushed this commit to bring the contact tests into compliance with the new displacements string, and, after this commit (hopefully) passes the tests, the commit should be ready for final review.  Thanks!

File: modules/combined/tests/fieldsplit_contact/2blocks3d.i [hunk 5]
```diff
@@ -100,6 +105,10 @@
     formulation = kinematic
     system = constraint
     normal_smoothing_distance = 0.1
+    order = FIRST
```
Comment (bwspenc) [inline modules/combined/tests/fieldsplit_contact/2blocks3d.i]: You shouldn't need to specify order or disp_* here (and you'll get a deprecation warning about the displacements).

**Review by bwspenc — CHANGES_REQUESTED** [review]
Just fix that issue with the contact options, and this will be good to go.

**Comment by sapitts** [comment]
@bwspenc this should be fixed now.  Thanks for the review

**Comment by sapitts** [comment]
third time (*i) is a charm, now that the tests have passed?  Should have all of the unnecessary disp_x removed from contact and density, finally, fingers crossed

**Review by bwspenc — APPROVED** [review]
